apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/ApmServerConfigurationSource.java

```diff
@@ -191,7 +191,7 @@ private String tryFetchConfig(ConfigurationRegistry configurationRegistry, HttpU
                 reader.startObject();
                 config = MapConverter.deserialize(reader);
                 configurationRegistry.reloadDynamicConfigurationOptions();
-                logger.info("Received new configuration from APM Server: {}", config);
+                logger.debug("Received new configuration from APM Server: {}", config);
```

Comment:
Why did the server not respond with `304 Not Modified`? Is there something wrong with the etag?